package org.example.refactor_under_test;

public class UserNotFoundException extends Exception {
}
